src/otx/v2/adapters/torch/mmengine/engine.py

```diff
@@ -606,7 +607,7 @@ def export(
             backend_config = {
                 "type": "openvino",
                 "model_inputs": [{"opt_shapes": {"input": [1, 3, 224, 224]}}],
-                "input_metas": {"mode": "predict"},
+                "input_metas": {"mode": "predict", "data_samples": [BaseDataElement()]},
```

Comment:
Could you let me know why we need to specify `data_samples`?

Reply:
Some of models requires some data classes for metainfo during export even if they are empty classes.